src/style.css

```diff
@@ -1 +1,105 @@
 /* add styles here */
+* {
+  box-sizing: border-box;
+  border: 0;
+
+  font-weight: 300;
+}
+
+.bar {
+  position: relative;
+  background: #ffff;
+
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  height: 70px;
+  width: auto;
```

Comment:
Avoid fixing the container size if there is no such requirement. Let the content size dictate the width of the bar instead of setting it to auto.